### PR TITLE
Adds Halloween Vinyl to the bar juke during halloween.

### PR DESCRIPTION
**Pull request by Kharkov72**
[sound]
Rattle me BONES.
This isn't the full juke update, we're just still working on the other playlists (Greyshit, jewvox, maint, spess, etc), and considering today is October 1st, we should probably get this into your hands, yeah?
This also reorganizes the superjuke a bit, so all the endgame songs are together, and gives the SCOTLAND vinyl a proper description, along with a few juke organizations into better categories.

This also makes Lil' Slugger its own vinyl, which starts in the bar and is the first vinyl in the list on both the bar juke and the superjuke.

Also, as always with juke PRs, leave this to pomf to merge so this and the juke repo pr get merged at the same time.

:cl:
 * rscadd: The Jukebox has successfully been haunted.

### CODE REVIEW

**Comment by DrSnips** [comment]
Can we get a song list with this?

**Comment by Kharkov72** [comment]
Untitled

**Comment by Probe1** [comment]
Waiting on the PR so I can merge the media server component first.

File: code/modules/media/jukebox.dm [hunk 1]
```diff
@@ -777,6 +778,13 @@ var/global/list/loopModeNames=list(
 		"upbeathypedancejam" = "Dance"
 	)
 
+/obj/machinery/media/jukebox/bar/New()
+	..()
+	var/MM = text2num(time2text(world.timeofday, "MM"))
+	if(MM == 10)
+		playlists.Add("halloween" = "Halloween")
```
Comment (DamianX) [inline code/modules/media/jukebox.dm:785]: `Runtime in jukebox.dm, line 785: illegal use of list2args() or named parameters`

Comment (Probe1) [reply code/modules/media/jukebox.dm:785]: Good catch.  Sorry for merging this without seeing it myself.

Comment (Kharkov72) [reply code/modules/media/jukebox.dm:785]: how should I fix this properly, sensei? @DamianX

Comment (ShiftyRail) [reply code/modules/media/jukebox.dm:785]: `playlists["halloween"] = "Halloween"`

**Pull request merged**